ParseStarterProject/src/com/parse/starter/LoginActivity.java

```diff
@@ -40,7 +40,7 @@ private void login() {
         String username = usernameEditText.getText().toString().trim();
         String password = passwordEditText.getText().toString().trim();
 
-        if(username.length() != 0 && password.length() != 0) {
+        if(username.length() != 0 && password.length() != 0) {  //requires user and pass not empty
```

Comment:
I realize I've also gotta do this for the RegisterActivity.  Any thoughts on how to format the appended message so it displays the minimum number of chars?